### PR TITLE
Add custom llm tool in my tools package

### PR DESCRIPTION
**Pull request by jiazengcindy**
# Description

Add custom llm tool in my tools package.

# All Promptflow Contribution checklist:
- [X] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [X] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by dorisjoy — DISMISSED** [review]

File: examples/tools/tool-package-quickstart/my_tool_package/tools/tool_with_custom_llm_type.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+from promptflow.connections import CustomConnection
+from promptflow.contracts.types import PromptTemplate
+
+
+@tool
+def my_tool(connection: CustomConnection, prompt: PromptTemplate, **kwargs) -> str:
+    # Replace with your tool code, customise your own code to handle and use the prompt here.
+    # Usually connection contains configs to connect to an API.
+    # Not all tools need a connection. You can remove it if you don't need it.
+    message = Template(prompt, trim_blocks=True, keep_trailing_newline=True).render(**kwargs)
```
Comment (chjinche) [inline examples/tools/tool-package-quickstart/my_tool_package/tools/tool_with_custom_llm_type.py:12]: `message`->`rendered_prompt`

File: examples/tools/tool-package-quickstart/my_tool_package/yamls/tool_with_custom_llm_type.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+my_tool_package.tools.tool_with_custom_llm_type.my_tool:
+  name: Custom LLM Tool
```
Comment (chjinche) [inline examples/tools/tool-package-quickstart/my_tool_package/yamls/tool_with_custom_llm_type.yaml:2]: Example Custom LLM Tool

Comment (jiazengcindy) [reply examples/tools/tool-package-quickstart/my_tool_package/yamls/tool_with_custom_llm_type.yaml]: Changed, thanks.

File: examples/tools/tool-package-quickstart/my_tool_package/tools/tool_with_custom_llm_type.py (hunk 1 above)
Comment (jiazengcindy) [reply examples/tools/tool-package-quickstart/my_tool_package/tools/tool_with_custom_llm_type.py]: Changed, thanks.

**Review by dorisjoy — APPROVED** [review]

**Review by chjinche — APPROVED** [review]

**Pull request merged**